llama_stack/apis/telemetry/telemetry.py

```diff
@@ -123,10 +137,67 @@ class StructuredLogEvent(EventCommon):
 ]
 
 
+@json_schema_type
+class EvalTrace(BaseModel):
+    session_id: str
+    step: str
+    input: str
+    output: str
+    expected_output: str
+
+
+@json_schema_type
+class MaterializedSpan(Span):
```

Comment:
some possibilities for name:

- SpanWithChildren
- SpanTree
- SpanSubTree

Reply:
done.